src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,212 @@
+                wordCount += words.length;
+                charCount += line.codePoints().toArray().length;
+            }
+            Path path = Paths.get(filePath);
+            fileSize = Files.size(path);
+            lastModified = Files.getLastModifiedTime(path);
+        } catch (Exception x) {
+            printFileError(filePath);
+            printFileError(filePath);
+        }
+        System.out.println(String.join("\n", new String[] {
+                "File path: " + filePath,
+                "Number of lines: " + lineCount,
+                "Number of words: " + wordCount,
+                "Number of chars: " + charCount,
+                "File size: " + fileSize,
+                "Last modified: " + lastModified
+        }));
+    }
+
+    private static Map<String, String> getAllCommandsHelp() {
+        return new LinkedHashMap<>() {{
+                put("create", String.join("\n", new String[] {
+                        "- create [path] [file-name]",
+                        "Creates file if the path exists and there is no such file already.",
+                        " "
+                }));
+                put("read", String.join("\n", new String[] {
+                        "- read [path] [file-name]",
+                        "Reads a file and prints content to console, if the file and path exist.",
+                        " "
+                }));
+                put("info", String.join("\n", new String[] {
+                        "- info [path] [file-name]",
+                        "Prints brief info about the file: number of symbols, words, lines,",
+                        "date and time of last modification, size.",
+                        " "
+                }));
+                put("help", String.join("\n", new String[] {
+                        "- help [command]",
+                        "Prints all available commands and information for them.",
+                        " "
+                }));
+                put("exit", String.join("\n", new String[] {
+                        "- exit",
+                        "Aborts the application work.",
+                        " "
+                }));
+            }};
+    }
+
+    public static void printHelp() {
+        String help = String.join("\n",
+                getAllCommandsHelp().values().toArray(new String[] {}));
+        System.out.println(help);
+    }
+
+    public static void printHelp(String command) {
+        Map<String, String> allCommandsHelp = getAllCommandsHelp();
+        String commandHelp = allCommandsHelp.get(command);
+        if (commandHelp != null) {
+            System.out.println(commandHelp);
+        } else {
+            System.out.println("There is no description for '" + command + "' available.");
+        }
+    }
+
+    public static void printExit() {
+        System.out.println("Exit. That's all, folks...");
+    }
+
+    public static boolean isPresentDirAndFile(String fileDir, String fileName) {
```

Comment:
```suggestion
    public static boolean isDirOrFilePresent(String fileDir, String fileName) {
```
assuming u use || operator :)